src/beautifiers/uncrustify/index.coffee

```diff
@@ -38,16 +38,18 @@ module.exports = class Uncrustify extends Beautifier
         editor = atom.workspace.getActiveTextEditor()
         if editor?
           basePath = path.dirname(editor.getPath())
-          # Expand Home Directory in Config Path
-          configPath = expandHomeDir(configPath)
+          projectPath = atom.workspace.project.getPaths()[0]
```

Comment:
There does not appear to be a good way to detect which path is currently selected. See https://discuss.atom.io/t/get-the-active-project/29309
This will have to do for now.